controller/iteration_test.go

```diff
@@ -347,7 +348,7 @@ func (rest *TestIterationREST) TestFailUpdateIterationNotFound() {
 
 func (rest *TestIterationREST) TestFailUpdateIterationUnauthorized() {
 	// given
-	_, _, _, itr := createSpaceAndRootAreaAndIterations(rest.T(), rest.db)
+	_, _, _, _, itr := createSpaceAndRootAreaAndIterations(rest.T(), rest.db)
```

Comment:
this function `createSpaceAndRootAreaAndIterations` is a helper and it creates a logical context to perform actions related to WI.
Most test cases are not using that new area object because they are either creating it in the test case or those tests are related to iteration hence area is not needed.
I need to update those area tests to make use of this function but I will see it in different PR, not this one.
I am using those returned objects in this test https://github.com/fabric8-services/fabric8-wit/pull/1466/files#diff-98914f859911696f3cdd98dd289d00faR2422

And while writing this I came to know I do not need these lines https://github.com/fabric8-services/fabric8-wit/pull/1466/files#diff-98914f859911696f3cdd98dd289d00faR2428 - https://github.com/fabric8-services/fabric8-wit/pull/1466/files#diff-98914f859911696f3cdd98dd289d00faR2434
i will fix it soon.